Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -952,6 +955,25 @@ void UpdateBarTextColor()
 				_tabLayout.TabTextColors = colors;
 		}
 
+		protected virtual void UpdateBarSelectedTextColor()
+		{
+			AColor selectedColor;
+			AColor unselectedColor;
+
+			if (Element != null)
+			{
+				selectedColor = (Element.BarSelectedTextColor.IsDefault ? Element.SelectedTabColor : Element.BarSelectedTextColor).ToAndroid();
+				unselectedColor = (Element.BarTextColor.IsDefault ? Element.UnselectedTabColor : Element.BarTextColor).ToAndroid();
+			}
+			else
+			{
+				selectedColor = ((Color)TabbedPage.BarSelectedTextColorProperty.DefaultValue).ToAndroid();
```

Comment:
Do we need to do anything if the Element is null?

Reply:
I think we should set the default value if `Element` is null which will give us consistency across the platforms. Currently the platform sets the native default value which is not consistent across iOS, Android and UWP. 

Going from memory here, but I believe the text color defaults is as follows:
- iOS - blue
- Android - white
- UWP - black